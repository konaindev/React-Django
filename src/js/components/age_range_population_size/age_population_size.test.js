import AgeRangePopulationSize from "./index";
import renderer from "react-test-renderer";

describe("AgeRangePopulationSize", () => {
  it("renders correctly", () => {
    const props = {
      age_group: "18-24",
      color: "#41C100",
      market_size: 2794,
      segment_population: 10369
    };
    const tree = renderer
      .create(<AgeRangePopulationSize {...props} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
