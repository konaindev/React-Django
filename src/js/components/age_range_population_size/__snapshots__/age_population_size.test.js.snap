// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AgeRangePopulationSize renders correctly 1`] = `
<div
  className="age-range-population-size"
>
  <div
    className="age-range-population-size__chart"
  >
    <div
      className="VictoryContainer"
      style={
        Object {
          "height": "100%",
          "pointerEvents": "none",
          "position": "relative",
          "touchAction": "none",
          "width": "100%",
        }
      }
    >
      <svg
        aria-labelledby="victory-container-1-title victory-container-1-desc"
        height={400}
        role="img"
        style={
          Object {
            "height": "100%",
            "pointerEvents": "all",
            "width": "100%",
          }
        }
        viewBox="0 0 400 400"
        width={400}
      >
        <g>
          <path
            d="M1.2246467991473532e-14,-200A200,200,0,0,1,198.50730145938996,24.38957292186291L0,0Z"
            role="presentation"
            shapeRendering="auto"
            style={
              Object {
                "fill": "#41C100",
                "padding": 10,
                "stroke": "transparent",
                "strokeWidth": 1,
              }
            }
            transform="translate(200, 200)"
          />
          <path
            d="M198.50730145938996,24.38957292186291A200,200,0,1,1,-3.6739403974420595e-14,-200L0,0Z"
            role="presentation"
            shapeRendering="auto"
            style={
              Object {
                "fill": "#000000",
                "padding": 10,
                "stroke": "transparent",
                "strokeWidth": 1,
              }
            }
            transform="translate(200, 200)"
          />
          <text
            direction="inherit"
            dx={0}
            dy={4.97}
            x={365}
            y={54}
          >
            <tspan
              dx={0}
              style={
                Object {
                  "fill": "#252525",
                  "fontFamily": "'Gill Sans', 'Gill Sans MT', 'Seravek', 'Trebuchet MS', sans-serif",
                  "fontSize": 14,
                  "letterSpacing": "normal",
                  "padding": 20,
                  "stroke": "transparent",
                }
              }
              textAnchor="start"
              x={365}
            >
              18-24
            </tspan>
          </text>
          <text
            direction="inherit"
            dx={0}
            dy={4.97}
            x={35}
            y={346}
          >
            <tspan
              dx={0}
              style={
                Object {
                  "fill": "#252525",
                  "fontFamily": "'Gill Sans', 'Gill Sans MT', 'Seravek', 'Trebuchet MS', sans-serif",
                  "fontSize": 14,
                  "letterSpacing": "normal",
                  "padding": 20,
                  "stroke": "transparent",
                }
              }
              textAnchor="end"
              x={35}
            >
              2
            </tspan>
          </text>
        </g>
      </svg>
      <div
        style={
          Object {
            "height": "100%",
            "left": 0,
            "position": "absolute",
            "top": 0,
            "width": "100%",
            "zIndex": 99,
          }
        }
      >
        <svg
          height={400}
          style={
            Object {
              "height": "100%",
              "overflow": "visible",
              "width": "100%",
            }
          }
          viewBox="0 0 400 400"
          width={400}
        />
      </div>
    </div>
  </div>
  <div
    className="age-range-population-size__info"
  >
    <div
      className="age-range-population-size__age-group"
    >
      Ages 
      18-24
    </div>
    <div
      className="age-range-population-size__market-size"
    >
      2,794
    </div>
    <div
      className="age-range-population-size__segment"
    >
      Out of 
      10,369
    </div>
  </div>
</div>
`;
